### PR TITLE
Initial github actions lint workflow.

### PR DESCRIPTION
**Pull request by ulloaluis**
## Checklist
- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [ ] The linter/Karma presubmit checks have passed.
  - These should run automatically, but if not, you can manually trigger them locally using `python -m scripts.pre_commit_linter` and `python -m scripts.run_frontend_tests`.
- [ ] The PR is made from a branch that's **not** called "develop".
- [ ] The PR has an appropriate "PROJECT: ..." label (Please add this label for the first-pass review of the PR).
- [ ] The PR has an appropriate "CHANGELOG: ..." label (If you are unsure of which label to add, ask the reviewers for guidance).
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- [ ] The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).)
- [ ] The PR is **assigned** to an appropriate reviewer.
  - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer and don't tick this checkbox.
  - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue. Do not only request the review but also add the reviewer as an assignee.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Setup
```
Comment (kevinlee12) [inline .github/workflows/build.yml]: Let's rename this since it isn't actually setup :)

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: lint
```
Comment (kevinlee12) [inline .github/workflows/build.yml]: Out of curiosity, can you try also renaming this file to `lint.yml` to see if it'll trigger a run?

Comment (ulloaluis) [reply .github/workflows/build.yml]: A run was triggered with the file being named lint.yml. (called lint / lint)

Comment (kevinlee12) [reply .github/workflows/build.yml]: sweet, thanks for testing that out!

**Comment by kevinlee12** [comment]
Lastly, can you test to make sure that it fails when there are valid linting errors? After that, feel free to open a PR on the main repo. Thanks!

**Comment by kevinlee12** [comment]
@ulloaluis I'd say run 2 more test cases then go ahead and make a PR on the main repo :smile: Please also make another merge from develop since there was a recent fix to the bad patterns check. Thanks! Btw, Happy Thanksgiving! :turkey:

**Pull request merged**